app/Actions/CreateDatabase.php

```diff
@@ -25,25 +25,36 @@ public function __invoke()
             return;
         }
 
-        if (! $this->mysqlExists()) {
-            return "MySQL does not seem to be installed. Skipping new database creation.";
-        }
+        app('console-writer')->logStep('Creating database');
+
+        $this->abortIf(! $this->mysqlExists(), "I can't create a database for your project because MySQL does not seem to be installed.");
 
-        $this->logStep('Creating database');
+        if (! $this->mysqlServerRunning()) {
+            app('console-writer')->ignoreVerbosity()->warn('Skipping database creation');
+            app('console-writer')->ignoreVerbosity()->warn("I can't create a database for your project because your MySQL server does not seem to be running.");
+            return;
+        }
```

Comment:
Can we merge these two into a single check, shaped like the second block, and just update the `if` to check for either condition, and update the message to say "because you don't have MySQL installed or it's not running"

Reply:
Done.